src/Microsoft.DotNet.Darc/tests/Microsoft.DotNet.DarcLib.Tests/VirtualMonoRepo/VmrPatchHandlerTests.cs

```diff
@@ -331,7 +334,16 @@ public async Task CreatePatchesWithSubmoduleWithoutChangesTest()
             Times.Never);
 
         patches.Should().ContainSingle();
-        patches.Single().Should().Be(new VmrIngestionPatch(expectedPatchName, string.Empty));
+        patches.Single().Should().Be(new VmrIngestionPatch(expectedPatchName, IndividualRepoName));
+
+        _dependencyTracker.Verify(x => x.UpdateSubmodules(It.IsAny<List<SubmoduleRecord>>()), Times.Exactly(1));
```

Comment:
I think this does not really verify anything since you then verify the same below but more strict

Reply:
I'd remove this one and keep the one below only

Reply:
Oooh, I get it now, you verify there are no other calls?

Reply:
There's also `VerifyNoOtherCalls()` but not sure it will help in this case since you are not setting up the calles using `Setup().Verifiable()`.

But you can give it a try.

Reply:
That was the idea, I will give it a try